### ISSUE TITLE
WCAG 2.2 Violation - There are instances of illogical reading order present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#5]
### Issue

There are instances of illogical reading order present within the interface (where the visual order of elements doesn't match the source order/reading order).

#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [1.3.2 Meaningful Sequence](https://www.w3.org/TR/WCAG22/#meaningful-sequence)

### Instances

1. #40 

### User Impact

Some users - such as sighted screen reader users - may be disoriented by the discrepancy between the visual order and the reading order.

#### Further information:

1. [The "Intent" section of "Understanding Success Criterion 1.3.2: Meaningful Sequence | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence.html#intent)

#### Using your browser's developer tools:

1. Take note of the visual order of the elements on the page.
2. Inspect the various element's using your browser's developer tools, taking note of the fact that the elements appear in a different order in the HTML (source) than they do visually on-screen.

#### Using a screen reader (such as VoiceOver, NVDA, JAWS, or TalkBack):

1. Activate the screen reader.
2. Navigate through the elements in question on the page, taking note of the fact that the elements are announced in a different order than they are presented visually.

### How To Fix The Issue

- Update the HTML source order (or the page's styles) so that the visual order matches the reading order (source order).
- (For disclosure controls only) Ensure that the toggled content appears immediately after the disclosure control in the source order.

#### WCAG 2.2 Conformant Code Example

Not available/not applicable.
